### DISCUSSION TITLE
SKIN menu skoda octavia 3 vrs 2016

### DISCUSSION THREAD

**Discussion by damirb08** [Mr-MIBonk/M.I.B._More-Incredible-Bash#455] [Q&A]
Hello, I need help. I had the need to touch something on the green menu and I managed to block the multimedia. I went to the skins menu (I thought I could change the colour of the backlight), in that menu I changed the parameter from 2 to 4 (it goes from 0-19). After the reboot the multimedia gets stuck on the logo screen I only have a VRS image and that's it. In the background, I hear the radio, phone, etc., but I can't do anything on multimedia. the reversing camera also works for me, but I have no options. Is there a solution to return that skin from 4 to 2 so that I don't touch anything anymore :). Sorry for my English.

SW TRAIN: MHI2_ER_SKG13_P4518   Columbus Harman  MIB2.5 9.2"

**Comment by HK7500** [comment]
Hello
The same thing happened to me in a VW. Any solution? 
Regards

**Reply by bee1850** [reply]
Any updates?

**Comment by bee1850** [comment]
I fixed it on my unit. Please remind me later today. I'll send the solution that worked for me.

**Reply by bee1850** [reply]
Downloaded FW from mibsolutions, then edited aio_special file in apps to change skin to 0 instead of 1 (to make sure, idk if i need this), then i plugged the contents which are on a sd card now in SD1 and connected to unit via telnet. Executed Script (mount -uw /net/mmx/fs/sda0; /net/mmx/fs/sda0/apps/aio_special -all) and Unit rebooted.
